tests/semanticdb/expect/i9782.expect.scala

```diff
@@ -0,0 +1,20 @@
+// LazyRef
+trait Txn/*<-_empty_::Txn#*/[T/*<-_empty_::Txn#[T]*/ <: Txn/*->_empty_::Txn#*/[T/*->_empty_::Txn#[T]*/]]
+
+trait Elem/*<-_empty_::Elem#*/[T/*<-_empty_::Elem#[T]*/ <: Txn/*->_empty_::Txn#*/[T/*->_empty_::Elem#[T]*/]]
+
+trait Obj/*<-_empty_::Obj#*/[T/*<-_empty_::Obj#[T]*/ <: Txn/*->_empty_::Txn#*/[T/*->_empty_::Obj#[T]*/]] extends Elem/*->_empty_::Elem#*/[T/*->_empty_::Obj#[T]*/]
+
+trait Copy/*<-_empty_::Copy#*/[In/*<-_empty_::Copy#[In]*/ <: Txn/*->_empty_::Txn#*/[In/*->_empty_::Copy#[In]*/], Out/*<-_empty_::Copy#[Out]*/ <: Txn/*->_empty_::Txn#*/[Out/*->_empty_::Copy#[Out]*/]] {
+  def copyImpl/*<-_empty_::Copy#copyImpl().*/[Repr/*<-_empty_::Copy#copyImpl().[Repr]*/[~ <: Txn[~]] <: Elem[~]](in/*<-_empty_::Copy#copyImpl().(in)*/: Repr/*->_empty_::Copy#copyImpl().[Repr]*/[In/*->_empty_::Copy#[In]*/]): Repr/*->_empty_::Copy#copyImpl().[Repr]*/[Out/*->_empty_::Copy#[Out]*/]
```

Comment:
it looks like here that there are no occurrences for the parameters of `Repr`

Reply:
although looking at the tree it seems maybe something else is hiding the occurrence:
```scala
-- Info: /Users/jamie/Workspace/dotty/tests/semanticdb/expect/i9782.scala:9:6 --
9 |  def copyImpl[Repr[~ <: Txn[~]] <: Elem[~]](in: Repr[In]): Repr[Out]
  |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |here is a defdef tree repr: DefDef(copyImpl,List(List(TypeDef(Repr,LambdaTypeTree(List(TypeDef(~,TypeBoundsTree(TypeTree[TypeRef(ThisType(TypeRef(NoPrefix,module class scala)),class Nothing)],AppliedTypeTree(Ident(Txn),List(Ident(~))),EmptyTree))),TypeBoundsTree(TypeTree[TypeRef(ThisType(TypeRef(NoPrefix,module class scala)),class Nothing)],AppliedTypeTree(Ident(Elem),List(Ident(~))),EmptyTree)))), List(ValDef(in,AppliedTypeTree(Ident(Repr),List(Ident(In))),EmptyTree))),AppliedTypeTree(Ident(Repr),List(Ident(Out))),EmptyTree)
```

Reply:
this could be addressed separately

Reply:
Oh, right. I think we can fix it in this PR, I'll take a look :)

Reply:
I found type params are going to this branch:
https://github.com/lampepfl/dotty/blob/28a671a71a3a5f6627a2db0c9cc5d9c0ec6218a2/compiler/src/dotty/tools/dotc/semanticdb/ExtractSemanticDB.scala#L168-L170

and `excludeChildren` is true if the parameter is HKTypeLambda, and that's why those symbols are missing.